csharp-basics/exercises/Tests/Arithmetics/Exercise6/CozaLozaWoza/CozaLozaWoza/CozaLozaWoza.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace CozaLozaWoza
+{
+    public class CozaLozaWoza : ICozaLozaWoza
+    {
+        public int Do_DecidingMethodTillConditionIsMet(int number)
+        {
+            while (number <= 110)
+                DecidingIfNumberIsChangedInto_3_5_7_AndWhenToMakeNewLine(number);
+
+            return number;
+        }
+
+        public string DecidingIfNumberIsChangedInto_3_5_7_AndWhenToMakeNewLine(int number)
```

Comment:
Ko šī metode dara? šis noteikti nav labs nosaukums. ConvertToCozaLozaWozaElement, piemēram.